test/form/samples/internal-conflict-resolution/_expected/amd.js

```diff
@@ -1,15 +1,15 @@
 define(function () { 'use strict';
 
-	var bar$1 = 42;
+	var bar = 42;
```

Comment:
Note these changes are a result of running deshadowing in execution order (due to it running on bundle.orderedModules over graph.modules). There are no other repercussions of this approach though.

Reply:
Reverted - please ignore.